README.md

```diff
@@ -18,19 +18,72 @@ We are going to track some new features/additions here so that it is quick and e
 - we've renamed the distributed .env files to ```switchboard.env.dist``` and ```frontend.env.dist```. This ensures that your local
   configuration doesn't get blown away when you pull changes from the repo. (We still use ```switchboard.env``` and ```frontend.env```
   for the config, it just means that new clones of the repo require the users to copy / rename the dist files)
-
 - we have added an extra `switchboard.env` called `CANARY_IPINFO_API_KEY`. This allows you to use your ipinfo.io api key if you
   want to (keep in mind ipinfo.io does have a free tier of up to 1000 requests a day).
-
 - we now have slack support. When you supply a webhook, you simply supply your slack webhook url. (Thanks to @shortstack).
-
 - we have added a new environment variable to `frontend.env` called `CANARY_AWSID_URL` which allows you to specify a private or
   different url for the AWS ID token. This means you can easily change between accounts. (2018-10-17)
 
+
+
+## Domain name / DNS configuration
+
+You register a domain name using a "registrar". This "registrar" has a communication channel with the "registry" which is the entity managing the TLD (`.com` for example).
+
+The registry operate some DNS servers that list domain names available into it's TLD, with the corresponding "next" DNS servers in the chain to send requests to for further records inside this domain zone.
+
+You configure in admin panel of you registrar, what DNS servers you want the registrar to "push" into the DNS servers managed by the registry.
```

Comment:
I dont think this is necessary. Although the explanation is nice, I feel it may confuse folks. We don't need to give too much details in our the DNS at that point works.

Reply:
I think people may get confused because we end with 2 differents "NS configurations".

- Registry level  (  j.gtld-servers.net )
```
canarytokens.com.		IN	NS	ns2.canarytokens.org.
canarytokens.com.		IN	NS	ns1.canarytokens.org.
```
- DNS level    ( ns2.canarytokens **.org.** )
```
canarytokens.com.		IN	NS	ns1.canarytokens.com.
```
So at one place we set **ns2**.canarytokens **.org.**   and in another we have **ns1**.canarytokens **.com.**.
Quite confusing  :)

I feel that if we don't explain a bit about those 2 levels, and where to configure the first one (at registrar) people may remains confused.

So I'm thinking  that it may be useful to explain a bit the first NS setup (registry DNS) can (must) be set (only) into the registrar configuration.. and not into their zone content configuration.

I fully agree we go quite "deep" into something not specific to the canarytokens part, but I have seen into the past issues few about this DNS setup.. So this may be required to go a bit deeper into the exemples.. ?

Reply:
Let me check this for you and get back to you. Thanks for taking the time